dist/parser-html.js

```diff
@@ -1 +1,3287 @@
.$LBRACE),this._endToken([]),this._expansionCaseStack.push(i.EXPANSION_FORM_START),this._beginToken(i.RAW_TEXT,this._getLocation());var e=this._readUntil(Dt.$COMMA);this._endToken([e],this._getLocation()),this._requireCharCode(Dt.$COMMA),this._attemptCharCodeUntilFn(v),this._beginToken(i.RAW_TEXT,this._getLocation());var t=this._readUntil(Dt.$COMMA);this._endToken([t],this._getLocation()),this._requireCharCode(Dt.$COMMA),this._attemptCharCodeUntilFn(v)}},{key:"_consumeExpansionCaseStart",value:function(){this._beginToken(i.EXPANSION_CASE_VALUE,this._getLocation());var e=this._readUntil(Dt.$LBRACE).trim();this._endToken([e],this._getLocation()),this._attemptCharCodeUntilFn(v),this._beginToken(i.EXPANSION_CASE_EXP_START,this._getLocation()),this._requireCharCode(Dt.$LBRACE),this._endToken([],this._getLocation()),this._attemptCharCodeUntilFn(v),this._expansionCaseStack.push(i.EXPANSION_CASE_EXP_START)}},{key:"_consumeExpansionCaseEnd",value:function(){this._beginToken(i.EXPANSION_CASE_EXP_END,this._getLocation()),this._requireCharCode(Dt.$RBRACE),this._endToken([],this._getLocation()),this._attemptCharCodeUntilFn(v),this._expansionCaseStack.pop()}},{key:"_consumeExpansionFormEnd",value:function(){this._beginToken(i.EXPANSION_FORM_END,this._getLocation()),this._requireCharCode(Dt.$RBRACE),this._endToken([]),this._expansionCaseStack.pop()}},{key:"_consumeText",value:function(){var e=this._getLocation();this._beginToken(i.TEXT,e);var t=[];do{this._interpolationConfig&&this._attemptStr(this._interpolationConfig.start)?(t.push(this._interpolationConfig.start),this._inInterpolation=!0):this._interpolationConfig&&this._inInterpolation&&this._attemptStr(this._interpolationConfig.end)?(t.push(this._interpolationConfig.end),this._inInterpolation=!1):t.push(this._readChar(!0))}while(!this._isTextEnd());this._endToken([this._processCarriageReturns(t.join(""))])}},{key:"_isTextEnd",value:function(){if(this._peek===Dt.$LT||this._peek===Dt.$EOF)return!0;if(this._tokenizeIcu&&!this._inInterpolation){if(T(this._input,this._index,this._interpolationConfig))return!0;if(this._peek===Dt.$RBRACE&&this._isInExpansionCase())return!0}return!1}},{key:"_savePosition",value:function(){return[this._peek,this._index,this._column,this._line,this.tokens.length]}},{key:"_readUntil",value:function(e){var t=this._index;return this._attemptUntilChar(e),this._input.substring(t,this._index)}},{key:"_restorePosition",value:function(e){this._peek=e[0],this._index=e[1],this._column=e[2],this._line=e[3];var t=e[4];t<this.tokens.length&&(this.tokens=this.tokens.slice(0,t))}},{key:"_isInExpansionCase",value:function(){return this._expansionCaseStack.length>0&&this._expansionCaseStack[this._expansionCaseStack.length-1]===i.EXPANSION_CASE_EXP_START}},{key:"_isInExpansionForm",value:function(){return this._expansionCaseStack.length>0&&this._expansionCaseStack[this._expansionCaseStack.length-1]===i.EXPANSION_FORM_START}}]),e}();function v(e){return!Dt.isWhitespace(e)||e===Dt.$EOF}function _(e){re
```

Comment:
is this instruction invalid on IE11 then?

e.g.

![image](https://user-images.githubusercontent.com/574210/66996105-659a8500-f0d0-11e9-835c-d56c720cf231.png)

Reply:
No, it acts the same as in Chrome. I was able to find that this error can occur because of the babel code transformation but there was no information how to fix it correctly